updater/lib/dependabot/file_fetcher_command.rb

```diff
@@ -250,5 +282,10 @@ def github_connectivity_client(job)
         }
       })
     end
+
+    def glob?(directory)
+      # We could tighten this up, but it's probably close enough.
+      directory.include?("*") || directory.include?("?") || (directory.include?("[") && directory.include?("]"))
```

Comment:
Can you explain how ? [ and ] correlate to a glob directory?

Reply:
It's a common supported syntax: https://en.wikipedia.org/wiki/Glob_(programming)#Syntax

```
irb(main):001> File.fnmatch?("[ab]c", "c", File::FNM_PATHNAME)
=> false
irb(main):002> File.fnmatch?("[ab]c", "ac", File::FNM_PATHNAME)
=> true
irb(main):003> File.fnmatch?("[ab]c", "bc", File::FNM_PATHNAME)
=> true
irb(main):004> File.fnmatch?("[ab]c", "cc", File::FNM_PATHNAME)
=> false
```